etc/DependencyInstaller.sh

```diff
@@ -34,6 +35,23 @@ _installCommon() {
     else
         pip3 install --no-cache-dir --user -U $pkgs
     fi
+
+    baseDir=$(mktemp -d /tmp/DependencyInstaller-orfs-XXXXXX)
```

Comment:
Why is this needed?  It is already set in the script elsewhere.

Reply:
I believe that `DependencyInstaller` from OpenROAD has this variable set once, but in ORFS each function has its own definitions of `baseDir`